modules/standard/DateTime.chpl

```diff
@@ -653,6 +697,9 @@ module DateTime {
 
   /* Return the daylight saving time offset */
   proc time.dst() {
+    if chpl_warnUnstable {
```

Comment:
Why are we warning about this under `--warn-unstable`? Does it return a new `tzinfo` or something?

Reply:
> Why are we warning about this under `--warn-unstable`? Does it return a new `tzinfo` or something?

Good catch. If these were using `tzinfo` there would have already been a warning when the object was initialized. It doesn't need an additional warning here or in `tzname`. Removed them.